packages/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+(modification: no type change headlines) and this project adheres to
+[Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.0.0] - 2020-08-25
```

Comment:
Great that we have a CHANGELOG here as well now! 🙂

No need to change this here for now, but I would suggest that we generally also move to this new process on the config libraries as well and add the CHANGELOG additions along the way but mark the CHANGELOG version entry as `UNRELEASED` and then doing a final release PR once everything is ready "unlocking" the CHANGELOG entry by adding the date, do the version bumps in the package.json files and update the respective READMEs.

Reply:
Right! I can split the file, for consistency sake to the other monorepo.

And I haven't used the `UNRELEASED` wording, because this is the actual release PR :)

Reply:
Done:
- Changelogs split into separate files
- `UNRELEASED` instead of 2.0.0